embedding-calculator/src/docs/find_faces_base64_post.yml

```diff
@@ -0,0 +1,91 @@
+tags:
+  - Core
+summary: 'Find faces in the given image in BASE64 format and return their bounding boxes.'
+description: 'Returns bounding boxes of detected faces on the image.'
+operationId: findFacesBase64Post
+consumes:
+  - multipart/form-data
+produces:
+  - application/json
+parameters:
+  - in: formData
```

Comment:
I believe it should be `body`, not `formData`

Reply:
Fixed